Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Core.cpp

```diff
@@ -287,6 +293,11 @@ namespace ScriptCanvas
 
     void SourceHandle::SanitizePaths()
     {
+        if (!m_relativePath.empty() && m_absolutePath.RelativePath() != m_relativePath)
```

Comment:
The `RelativePath` calls within this function needs to be removed.

That is only returning the Relative portion of the absolute path.

Reply:
This part is about to get removed by a competing PR that will have to merge over this actually.

Reply:
https://github.com/o3de/o3de/pull/10167  This one in particular, needs to go in first.